helm-chart/csi-driver/templates/get-linode-id.yaml

```diff
@@ -23,7 +23,7 @@ data:
       # Try to get ID using Metadata API with a 5-second timeout
       TOKEN=$(curl -sS -m 5 -X PUT -H "Metadata-Token-Expiry-Seconds: 3600" $META_API/token || true)
       if [ ! -z "$TOKEN" ]; then
-        ID=$(curl -sS -m 5 -H "Metadata-Token: $TOKEN" $META_API/instance -H "Accept: application/json" | grep -o '"id": [0-9]*' | cut -d' ' -f2 |  tr -d '\n' || true)
+        ID=$(curl -sS -m 5 -H "Metadata-Token: $TOKEN" $META_API/instance -H "Accept: application/json" | jq -r .id |  tr -d '\n' || true)
```

Comment:
context... where does this run and do we for sure have JQ there?

Reply:
this runs in the init container - which defaults to `alpine/k8s` https://hub.docker.com/r/alpine/k8s - which does have jq

Reply:
confirmed it exists 


```
$ docker run -it alpine/k8s:1.25.14 bash
dee1d4fe298c:/apps# which jq
/usr/bin/jq
```